gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -474,6 +476,8 @@ def assign_pops(
     max_mislabeled_training_samples: int = 50,  # TODO: Think about this parameter and add it to assign_population_pcs. Maybe should be a fraction? fraction per pop?
     n_pcs: int = 16,
     withhold_prop: float = None,
+    subpops: bool = False,
```

Comment:
Let's change n_pcs to PCs with default: `pcs=range(1, 17)` The reason is that for subpops we sometimes skip PCs, so we want to be able to specify the exact PCs we want to use